samples/hip_async_interop/kernel.h

```diff
@@ -0,0 +1,486 @@
+  0x80, 0x00, 0x05, 0x00, 0x0f, 0x00, 0x00, 0x00, 0xa4, 0x00, 0x00, 0x00,
+  0x9d, 0x00, 0x00, 0x00, 0xa3, 0x00, 0x00, 0x00, 0xac, 0x00, 0x05, 0x00,
+  0x8c, 0x00, 0x00, 0x00, 0xa5, 0x00, 0x00, 0x00, 0xa4, 0x00, 0x00, 0x00,
+  0x22, 0x00, 0x00, 0x00, 0xfa, 0x00, 0x04, 0x00, 0xa5, 0x00, 0x00, 0x00,
+  0x1c, 0x00, 0x00, 0x00, 0x1d, 0x00, 0x00, 0x00, 0xf8, 0x00, 0x02, 0x00,
+  0x1c, 0x00, 0x00, 0x00, 0x3d, 0x00, 0x06, 0x00, 0x05, 0x00, 0x00, 0x00,
+  0xa6, 0x00, 0x00, 0x00, 0x99, 0x00, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00,
+  0x10, 0x00, 0x00, 0x00, 0x3d, 0x00, 0x06, 0x00, 0x05, 0x00, 0x00, 0x00,
+  0xa7, 0x00, 0x00, 0x00, 0x9a, 0x00, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00,
+  0x10, 0x00, 0x00, 0x00, 0x85, 0x00, 0x05, 0x00, 0x05, 0x00, 0x00, 0x00,
+  0xa8, 0x00, 0x00, 0x00, 0x74, 0x00, 0x00, 0x00, 0xa7, 0x00, 0x00, 0x00,
+  0x0c, 0x00, 0x08, 0x00, 0x05, 0x00, 0x00, 0x00, 0xa9, 0x00, 0x00, 0x00,
+  0x01, 0x00, 0x00, 0x00, 0x2a, 0x00, 0x00, 0x00, 0x73, 0x00, 0x00, 0x00,
+  0xa6, 0x00, 0x00, 0x00, 0xa8, 0x00, 0x00, 0x00, 0x3e, 0x00, 0x05, 0x00,
+  0x8a, 0x00, 0x00, 0x00, 0xa9, 0x00, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00,
+  0x10, 0x00, 0x00, 0x00, 0xf9, 0x00, 0x02, 0x00, 0x1d, 0x00, 0x00, 0x00,
+  0xf8, 0x00, 0x02, 0x00, 0x1d, 0x00, 0x00, 0x00, 0xe0, 0x00, 0x04, 0x00,
+  0x90, 0x00, 0x00, 0x00, 0x90, 0x00, 0x00, 0x00, 0x91, 0x00, 0x00, 0x00,
+  0x80, 0x00, 0x05, 0x00, 0x0f, 0x00, 0x00, 0x00, 0x9c, 0x00, 0x00, 0x00,
+  0x9d, 0x00, 0x00, 0x00, 0xaa, 0x00, 0x00, 0x00, 0xad, 0x00, 0x05, 0x00,
+  0x8c, 0x00, 0x00, 0x00, 0xac, 0x00, 0x00, 0x00, 0x9d, 0x00, 0x00, 0x00,
+  0x90, 0x00, 0x00, 0x00, 0xfa, 0x00, 0x04, 0x00, 0xac, 0x00, 0x00, 0x00,
+  0x19, 0x00, 0x00, 0x00, 0x18, 0x00, 0x00, 0x00, 0xf8, 0x00, 0x02, 0x00,
+  0x1e, 0x00, 0x00, 0x00, 0x3d, 0x00, 0x06, 0x00, 0x05, 0x00, 0x00, 0x00,
+  0xad, 0x00, 0x00, 0x00, 0x94, 0x00, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00,
+  0x10, 0x00, 0x00, 0x00, 0x46, 0x00, 0x05, 0x00, 0x10, 0x00, 0x00, 0x00,
+  0xae, 0x00, 0x00, 0x00, 0x15, 0x00, 0x00, 0x00, 0x26, 0x00, 0x00, 0x00,
+  0x3e, 0x00, 0x05, 0x00, 0xae, 0x00, 0x00, 0x00, 0xad, 0x00, 0x00, 0x00,
+  0x02, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00, 0x00, 0xf9, 0x00, 0x02, 0x00,
+  0x1f, 0x00, 0x00, 0x00, 0xf8, 0x00, 0x02, 0x00, 0x1f, 0x00, 0x00, 0x00,
+  0xfd, 0x00, 0x01, 0x00, 0x38, 0x00, 0x01, 0x00, 0x36, 0x00, 0x05, 0x00,
+  0x0e, 0x00, 0x00, 0x00, 0xaf, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x11, 0x00, 0x00, 0x00, 0x37, 0x00, 0x03, 0x00, 0x0f, 0x00, 0x00, 0x00,
+  0xb0, 0x00, 0x00, 0x00, 0x37, 0x00, 0x03, 0x00, 0x10, 0x00, 0x00, 0x00,
+  0xb1, 0x00, 0x00, 0x00, 0x37, 0x00, 0x03, 0x00, 0x10, 0x00, 0x00, 0x00,
+  0xb2, 0x00, 0x00, 0x00, 0xf8, 0x00, 0x02, 0x00, 0xb3, 0x00, 0x00, 0x00,
+  0x39, 0x00, 0x07, 0x00, 0x0e, 0x00, 0x00, 0x00, 0xb4, 0x00, 0x00, 0x00,
+  0x12, 0x00, 0x00, 0x00, 0xb0, 0x00, 0x00, 0x00, 0xb1, 0x00, 0x00, 0x00,
+  0xb2, 0x00, 0x00, 0x00, 0xfd, 0x00, 0x01, 0x00, 0x38, 0x00, 0x01, 0x00
+};
+unsigned int KernelSpirVLength = 5796;
```

Comment:
Again, seems excessive when you could just have a SPIR-V file and use it for both OpenCL and Level0

Reply:
Seemes like you already have a `kernel.spv`?